MusicCatalog/src/utils/checkPrice.js

```diff
@@ -0,0 +1 @@
+export const checkPrice = price => (price>0 ? price + ' $' : 'FREE');
```

Comment:
You don't need to create a separate file for each helper. Please, create an index.js file and move all utils functions here.